### ISSUE TITLE
Where can i find the original code for faster_rcnn_inception_resnet_v2_atrous_oidv2 for OpenImage?

### ISSUE THREAD

**Issue by username123062** [tensorflow/models#7993]
I only found the pb file for the faster_rcnn_inception_resnet_v2_atrous_oidv2 for OpenImage. So, where can i find the original code for faster_rcnn_inception_resnet_v2_atrous_oidv2 for OpenImage? Thanks very much

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by KieranLitschel** [comment]
/models/research/object_detection/models/keras_models/inception_resnet_v2_test.py describes the changes to the original Keras model, and the original Keras model is declared here https://github.com/keras-team/keras-applications/blob/master/keras_applications/inception_resnet_v2.py

**Comment by username123062** [comment]
Thanks. Besides the inception_resnet_v2.py, where can I find the code of the input and loss function about the dataset "OpenImage" using the inception_resnet_v2.py?

**Comment by username123062** [comment]
@pkulzc Hi, I found that you are one of the contributors of 'models/research/object_detection/g3doc/detection_model_zoo.md'. So can you help me to solve the above problem? Thanks a lot.

Because I am a beginner with OpenImage V2, I want to know the input data file (including how to use the image path, image bounding box labels, etc). I hope you can help me. Thanks again.

**Comment by username123062** [comment]
I still cannot found the "oid_bbox_trainable_train.record" file, could someone help me? Thanks
